.github/workflows/cd-teardown.yml

```diff
@@ -6,20 +6,24 @@ env:
   CI: true
 jobs:
   unpublish-npm:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
     if: github.event.ref_type == 'branch'
     steps:
     - name: Prepare for unpublication from npm
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v2.1.4
```

Comment:
Hmm... I thought Snyk was enabled for solid-client as well - I haven't seen any PR there yet, but I assumed that was because Dependabot was ahead of it.

In any case, you can just enable Dependabot for _just_ GitHub Actions, which I assume Snyk doesn't keep updated.

Reply:
Looks like dependabot is configured here anyways. I've pushed an update for .github/dependabot.yml to include actions.